license_demo/WORKSPACE

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+workspace(name = "demo")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_license",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_license/releases/download/0.0.1/rules_license-0.0.1.tar.gz",
+    ],
+    sha256 = "4865059254da674e3d18ab242e21c17f7e3e8c6b1f1421fffa4c5070f82e98b5",
+)
+
+# You must expose your own license declarations as a repository so the rules
+# generated in @maven can point to them. Otherwise you would have to generate
+# the declarations in @maven.
+local_repository(
+  name = "my_compliance",
+  path = "compliance",
+)
+
+
+# Point to parent for testing. You would use http_archive in real life.
+http_archive(
+    name = "rules_jvm_external",
+    # path = "..",
+    urls = ["file:///tmp/rje.tar"],
```

Comment:
How did you generate this `.tar`? I ended up just using `git_repository()` workspace rule pointing at this branch as a work-around

Reply:
Tar from the command line. This is just to get something to demo, not what we would really do.